### ISSUE TITLE
possible bug in provisioning script

### ISSUE THREAD

**Issue by joshsunshine** [schmerl/LLStaging#19]
```==> default: ./setup-cp1.sh: line 8: /opt/ros/indigo/setup.bash: No such file or directory
==> default: ./setup-cp1.sh: line 10: catkin_init_workspace: command not found
==> default: ./setup-cp1.sh: line 12: catkin_make: command not found
```
when doing the vagrant install from a clean machine that's never seen this before. it looks like it's trying to run catkin in the host OS?

**Comment by joshsunshine** [comment]
this also happens if you run `setup` inside the guest machine:
```vagrant@vagrant-ubuntu-trusty-64:~$ ./setup-cp1.sh 
./setup-cp1.sh: line 8: /opt/ros/indigo/setup.bash: No such file or directory
./setup-cp1.sh: line 10: catkin_init_workspace: command not found
./setup-cp1.sh: line 12: catkin_make: command not found
vagrant@vagrant-ubuntu-trusty-64:~$ logout```

**Comment by joshsunshine** [comment]
in `vagrant provision` output:
```
==> default: /tmp/vagrant-shell: line 15: /opt/ros/indigo/setup.bash: No such file or directory
==> default: sudo: rosdep: command not found
==> default: sudo: rosdep: command not found
==> default: /tmp/vagrant-shell: line 18: rosdep: command not found
==> default: cp: 
==> default: cannot create regular file ‘/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/’
==> default: : No such file or directory
```

**Comment by joshsunshine** [comment]
This issues was resolved by updating to git version 2.11 and git-lfs version 1.5.5

**Comment by ivoysey** [comment]
git-lfs was silently not installing itself and downloading almost but not quite right versions of the files.  it's a little odd, but we seem to require very new git and git-lfs in order to deploy old ubuntu, ros indigo, etc.
